index.js

```diff
@@ -3,190 +3,226 @@
  *
  * High-level audio container
  */
-'use strict';
+'use strict'
+
+const Emitter = require('events')
```

Comment:
I've heard that this bundles a little big from other packages.

After looking at bundle sizes for this, maybe we could consider [mitt](https://github.com/developit/mitt) or [nanobus](https://github.com/yoshuawuyts/nanobus) (used in choo)?